app/controllers/application_controller.rb

```diff
@@ -16,5 +16,14 @@ def configure_permitted_parameters
                          :password_confirmation,
                          :avatar)
     end
+    devise_parameter_sanitizer.permit(:account_update) do |user_params|
+      user_params.permit({ roles: [] },
```

Comment:
We should combine this with the one above. keys = []

Then reference that once inside each permit.